hazenlib/ghosting.py

```diff
@@ -223,25 +244,28 @@ def get_ghosting(dcm, plotting=False) -> dict:
         img = cv.rectangle(img.copy(), (x1, y1), (x2, y2), (255, 0, 0), 1)
 
         ax.imshow(img)
+        fig.savefig(f'{report_path}.png')
```

Comment:
When I run `hazen ghosting ... --report` it outputs a file called `True.png`. I think it's because you have used `report_path` here, which = True or False.

On a related note, the function always outputs `True.png`, even without the `--report` option. Is that supposed to happen?